executor/compiler.go

```diff
@@ -363,6 +363,10 @@ func getDbFromResultNode(resultNode ast.ResultSetNode) []string { // may have du
 	return dbLabels
 }
 
+func init() {
+	plannercore.GetStmtLabel = GetStmtLabel
```

Comment:
1、the init of ```plannercore.GetStmtLabel```  should be put at the ```planner/core```. It is direct.

2、``` GetStmtLabel``` should be moved into ```ast```. It is a function of ast. we should reduce the complexity of dependencies.

Reply:
You are right. I move this function to `ast` now.